assets/css/app.css

```diff
@@ -0,0 +1,203 @@
+html {
+ height: 100%;
+}
+
+body {
+ color: #333;
+ font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+ -webkit-font-smoothing: antialiased;
+ height: 100%;
+}
+
+main {
+ height: 70%;
+ min-height: 70%;
+}
+
+footer {
+ height: 29%;
+ margin-top: 2px;
+ min-height: 29%;
+}
+
+.container {
+ height: 100%;
+ margin: 0 auto;
+ min-height: 100%;
+ min-width: 1440px;
+ width: 1440px;
+}
+
+.hero-unit {
+ background: url("../images/your_verse_hero.jpg") 50% 100% no-repeat;
+ background-position: bottom center;
+ background-size: auto;
+ overflow-y: hidden;
+}
+
+.hero-unit-center {
+ -ms-align-items: center;
+ align-items: center;
+ display: flex;
+ flex-direction: column;
+ justify-content: center;
+ text-align: center;
+}
+
+.hero-unit-center .top {
+ padding-top: 11rem;
```

Comment:
The percentage might be better in this case 🤔 

As you can see on smaller screens the hero text is a tad bit too far down. Using a percentage would give better results.

Also, a great job that you tried to use `em` where you can! Keep at it. Should use it as opposed to px whenever you can. Which you have!

See image below to see what I am talking about with the padding!
![Screen Shot 2019-04-11 at 9 43 46 AM](https://user-images.githubusercontent.com/31193823/55963276-832df400-5c40-11e9-8b28-18638ee18773.png)